sdk/keyvault/azure-keyvault-keys/tests/keys_test_case.py

```diff
@@ -14,6 +15,11 @@ def setUp(self):
 
     def tearDown(self):
         super(KeyVaultTestCase, self).tearDown()
+        if self.is_live:
+            dirname = os.path.dirname(__file__)
+            seed_filename = os.path.abspath(os.path.join(dirname, "seed.txt"))
```

Comment:
Similarly, `join` will return an absolute path here because `dirname` is absolute.